distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -530,15 +540,20 @@ type Array
        in the input array.
 
        Arguments:
-       - predicate: A function that takes an element and returns a boolean
-         value.
+       - condition: A `Filter_Condition` or a function that takes an element and
+         returns a boolean value.
+
+       > Example
+         Splitting an array into elements that start with a prefix.
+
+             ["a", "b", "ax", "bx"].to_array.partition (Filter_Condition.Starts_With "a") == (Pair ["a", "ax"].to_array ["b", "bx"].to_array)
```

Comment:
```suggestion
             ["a", "b", "ax", "bx"].to_array.partition (Filter_Condition.Starts_With "a") == (Pair ["a", "ax"]["b", "bx"])
```

These `to_array` on the right should not be there. They are 'technically correct', because `["a", "ax"].to_array == ["a", "ax"]`, but `Array.partition` returns _vectors_ so IMO they are misleading.

Nevertheless they have to be there to satisfy the `VectorArraySignatureTest`. Finding some better heuristic for the test seems to be out of scope of this PR.